graveler/sstable/cache.go

```diff
@@ -53,21 +53,26 @@ func (i *item) Close() error {
 }
 
 func NewCache(p lru.ParamsWithDisposal, fs pyramid.FS, readerOptions sstable.ReaderOptions) Cache {
-	return NewCacheWithOpener(p,
-		func(namespace string, id string) (Item, error) {
-			file, err := fs.Open(namespace, id)
-			if err != nil {
-				return nil, fmt.Errorf("fetch %s from next tier: %w", id, err)
-			}
-
-			reader, err := sstable.NewReader(file, readerOptions)
-			if err != nil {
-				return nil, fmt.Errorf("open SSTable %s: %w", id, err)
-			}
-			return &item{reader}, nil
-		},
-		fs.Exists,
-	)
+	opener := makePyramidOpener(fs, readerOptions)
+	if p.Size == 0 {
+		return &noCache{opener: opener}
```

Comment:
Do you still want to add `noCache`?

Reply:
can remove it - needed something to work.  WDYT?